<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>325</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PTNG Workbench Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/workbench.qrc">
    <normaloff>:/images/goldings_gym_core_00.png</normaloff>:/images/goldings_gym_core_00.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>156</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="lafGroupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Look-and-feel</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="fontLabel">
      <property name="text">
       <string>Font:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFontComboBox" name="fontComboBox"/>
    </item>
    <item>
     <widget class="QSpinBox" name="fontSpinBox">
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>382</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>Project preferences</string>
   </property>
   <widget class="QLabel" name="wdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Default working directory:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wdLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="wdPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/workbench.qrc">
      <normaloff>:/images/folder_open_01.png</normaloff>:/images/folder_open_01.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="pdLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>128</y>
      <width>321</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>98</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Default project directory:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>126</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../res/workbench.qrc">
      <normaloff>:/images/folder_open_01.png</normaloff>:/images/folder_open_01.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res/workbench.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
